connect/src/main/protobuf/spark/connect/expressions.proto

```diff
@@ -146,6 +146,19 @@ message Expression {
     repeated string parts = 1;
   }
 
+  // An resolved attribute that can specify a reference (e.g. column) without needing a resolution
+  // by the analyzer.
+  message Attribute {
```

Comment:
Because for a LocalRelation, they are all resolved attributes. If you build UnresolvedAttribute, where to resolve? Its local relation without being populated into the catalog. Same for `Nullability` as it is self resolved so you need to tell that what is `Nullability`. Again Spark cannot resolve it given it is called `local`.

This usually happens when users write `Seq(a,b,c,e).toDF` I believe. This is a common usage for fast development.